common/cancellation_error_code.go

```diff
@@ -8,38 +8,36 @@ type CancellationErrorCode int
 
 const (
 
-	// No error.
-	// If CancellationReason is EndOfStream, CancellationErrorCode
+	// NoError if CancellationReason is EndOfStream, CancellationErrorCode
 	// is set to NoError.
 	NoError CancellationErrorCode = 0
 
-	// Indicates an authentication error.
+	// AuthenticationFailure indicates an authentication error.
 	// An authentication error occurs if subscription key or authorization token is invalid, expired,
 	// or does not match the region being used.
 	AuthenticationFailure CancellationErrorCode = 1
 
-	// Indicates that one or more recognition parameters are invalid or the audio format is not supported.
-
+	// BadRequest indicates that one or more recognition parameters are invalid or the audio format is not supported.
 	BadRequest CancellationErrorCode = 2
 
-	// Indicates that the number of parallel requests exceeded the number of allowed concurrent transcriptions for the subscription.
+	// TooManyRequests indicates that the number of parallel requests exceeded the number of allowed concurrent transcriptions for the subscription.
```

Comment:
Side note (as we discussed over IM): this is the first I've seen this linter rule in action and my initial reaction to it is very negative. It seems like it just needlessly clutters/duplicates the name and I'm struggling to imagine a reason for it that couldn't/shouldn't be accomplished by better tooling.

Not really about this PR, though: just may be worthwhile as a low-pri follow-up to see if this rule has good reasoning behind it. And if it doesn't to disable it across the project.